src/components/menuChooser.jsx

```diff
@@ -41,6 +41,7 @@ function MenuChooser() {
           img={NormalizationImg}
           title={"Writer Interphase App"}
           ChoosingAppArrow={ChoosingAppArrow}
+          DisplayContent={<div>Lorem ipsum dolor sit amet,<br/>consectetur adipiscing</div>}
```

Comment:
why are u passing `div` like this. It should be inside the component.

Reply:
`displayContent={"Lorem ipsum dolor sit amet,consectetur adipiscing"}`

Reply:
addressed